pypeit/spectrographs/spectrograph.py

```diff
@@ -1182,13 +1182,9 @@ def get_meta_value(self, inp, meta_key, required=False,
                             if ftype in self.meta[meta_key]['required_ftypes']:
                                 kerror = True
                     # Bomb out?
-                    # TODO Bombing out is not acceptable as we have discussed.
                     if kerror:
-                        #embed(header=utils.embed_header())
-                        msgs.warn('Required meta "{0}" did not load!'.format(meta_key)
-                                  + 'You may have a corrupt header.')
-                        #msgs.error('Required meta "{0}" did not load!'.format(meta_key)
-                        #           + 'You may have a corrupt header.')
+                        msgs.error('Required meta "{0}" did not load!'.format(meta_key)
```

Comment:
Same comment as above. This change was added to prevent a bad header crash. Until you demonstrate on a file with bad headers that pypeit_setup does not crash this needs to throw a warning and not bomb out.